tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -837,3 +837,108 @@ def _check_ssh_auth(user, expect_success=True):
             raise e
         else:
             logging.info(f"SSH access denied for user {user.alias}, as expected")
+
+
+@pytest.mark.parametrize(
+    "directory_type,directory_protocol,directory_certificate_verification",
+    [
+        ("SimpleAD", "ldap", False),
+        ("MicrosoftAD", "ldaps", True),
+    ],
+)
+@pytest.mark.usefixtures("os", "instance")
+def test_ad_integration_on_login_nodes(
+    region,
+    scheduler,
+    scheduler_commands_factory,
+    pcluster_config_reader,
+    directory_type,
+    directory_protocol,
+    directory_certificate_verification,
+    test_datadir,
+    directory_factory,
+    request,
+    store_secret_in_secret_manager,
+    clusters_factory,
+):
+    """
+    Verify AD integration works as expected.
+    In particular, it verifies that :
+    1. AD users can access to the head node, both with password and SSH key (if created);
```

Comment:
can access to the login nodes, not head node

Reply:
Good catch!
Fixing it..